deployer/deployer/src/test.rs

```diff
@@ -13,20 +13,14 @@ mod contract {
 #[test]
 fn test() {
     let env = Env::default();
-    let deployer_id = BytesN::from_array(&env, &[0; 32]);
-    env.register_contract(&deployer_id, Deployer);
-    let client = DeployerClient::new(&env, &deployer_id);
+    let client = DeployerClient::new(&env, &env.register_contract(None, Deployer));
 
     // Deploy contract using deployer, and include an init function to call.
     let salt = Bytes::from_array(&env, &[0; 32]);
-    let wasm: Bytes = contract::WASM.into_val(&env);
+    let wasm_hash = env.install_contract_wasm(contract::WASM);
```

Comment:
Could you move this line above the comment, and add its own comment signaling that it is being installed prior to deployment? I think pulling it out will help people understand that step clearer.

Reply:
Updated.